docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.8'
+
+networks:
+  k6:
+
+services:
+  prometheus:
+    image: prom/prometheus:v2.52.0
+    command:
+      - --web.enable-remote-write-receiver
+      - --enable-feature=native-histograms
+      - --config.file=/etc/prometheus/prometheus.yml
+    networks:
+      - k6
+    ports:
+      - "9090:9090"
+
+  alloy:
```

Comment:
If there is a specific reason for using it over the [OTLP receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver) built-in in Prometheus then it would be good to comment about it.

Reply:
Not really! I also wanted to try alloy, but I could investigate the following PRs using Prometheus. I will merge as is and open the follow-up task.